src/test/java/formflow/library/submissions/actions/CheckIndicatedContactMethodIsProvided.java

```diff
@@ -2,27 +2,34 @@
 
 import formflow.library.config.submission.Action;
 import formflow.library.data.FormSubmission;
-
+import java.util.ArrayList;
```

Comment:
Why is there a `CheckIndicatedContactMethodIsProvided` in both the library and [the starter app](https://github.com/codeforamerica/form-flow-starter-app/blob/fd10854d2e1cccdc56a05042848f3f8f211f6b9f/src/main/java/org/formflowstartertemplate/app/submission/actions/CheckIndicatedContactMethodIsProvided.java)?

Is it for testing? Would the starter app use this one in the library?

Reply:
This one is for testing the implementation of a CrossFieldValidation action. It matches the starter-app because that was the use case this was originally needed for so I just made the test for that specific use case. The thought was that the library should have a test for each style of action we have.

Reply:
Gotcha, makes sense.